src/main/webapp/search.js

```diff
@@ -93,6 +94,7 @@ function addVideoInfo(video) {
       showLoadingGif(SEARCH_ID_PREFIX);
       updateUIWithVideoContext(youtubeUrl, SEARCH_ID_PREFIX);
       displayOverallResults(youtubeUrl, SEARCH_ID_PREFIX);
+      enableLoadingButton('search-button');
```

Comment:
Question: No disableLoadingButton needed here? And no async needed?

Reply:
disable is done beforehand. This is the time whe everything is displayed and we are ready for the next searfch